src/Mod/Path/PathScripts/PathPost.py

```diff
@@ -135,6 +130,15 @@ def resolveFileName(self, job):
                 self.subpart += 1
             else:
                 filename = filename.replace("%s", "")
+        return filename
+
+    def resolveFileName(self, job):
+        path = PathPreferences.defaultOutputFile()
+        if job.PostProcessorOutputFile:
+            path = job.PostProcessorOutputFile
+        filename = self.processFileNameSubstitutions(job, path)
+        if filename is None:
+            return None
```

Comment:
If filename is `None`, it should return `None` instead? Hummm, just `return None` should work then?